adapters/base/src/main/java/org/apache/cassandra/sidecar/adapters/base/CassandraStorageOperations.java

```diff
@@ -201,4 +202,12 @@ public List<String> dataFileLocations()
         }
         return dataFileLocations;
     }
+
+    @Override
+    public void outOfRangeDataCleanup(@NotNull String keyspace, @NotNull String table, int concurrency)
+    throws IOException, ExecutionException, InterruptedException
+    {
```

Comment:
For consistency with the behavior in other methods in this class and generally good practice 
```suggestion
    {
        requireNonNull(keyspace, "keyspace must be non-null");
        requireNonNull(table, "table must be non-null");
```